benchmarks/src/main/scala/zio/IOShallowAttemptBenchmark.scala

```diff
@@ -130,7 +130,7 @@ class IOShallowAttemptBenchmark {
   def scalazShallowAttempt(): BigInt = {
     def throwup(n: Int): IO[ScalazError, BigInt] =
       if (n == 0) throwup(n + 1).fold[BigInt](_ => 50, identity)
-      else if (n == depth) IO.succeedLazy(1)
+      else if (n == depth) IO.succeed(1)
```

Comment:
This should become `effectTotal` in line with other effect types that delay execution in their benchmarks (e.g. monix `Task.eval`)

Reply:
:heavy_check_mark: